src/Tracy/Debugger/Debugger.php

```diff
@@ -324,7 +324,12 @@ public static function exceptionHandler(\Throwable $exception): void
 					. (isset($e)
 						? 'Unable to log error. You may try enable debug mode to inspect the problem.'
 						: 'Check log to see more info.')
-					. "\n");
+					. (!self::$productionMode
```

Comment:
Did you test it?

Reply:
I think now it is perfect. First call is production, second is debug.

![Snímek obrazovky 2021-06-09 v 20 04 29](https://user-images.githubusercontent.com/4738758/121406233-30564100-c95e-11eb-9ce2-c66bc0a01803.png)

Testing code:

```php
Debugger::$productionMode = false;
throw new \RuntimeException('Test for DG.');
```

Thanks.